packages/react-composites/src/composites/CallWithChatComposite/NotificationIcon.tsx

```diff
@@ -32,39 +31,37 @@ export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
   };
 
   return (
-    <Stack horizontalAlign="center" verticalAlign="center" styles={notificationIconContainerStyles}>
-      <CallWithChatCompositeIcon styles={notificationIconStyles(theme)} iconName="ControlBarButtonBadgeIcon" />
-      {renderNumber(chatMessagesCount)}
+    <Stack horizontalAlign="center" verticalAlign="center" styles={notificationIconContainerStyles(theme)}>
+      <Stack>{renderNumber(chatMessagesCount)}</Stack>
     </Stack>
   );
 };
 
+const notificationIconPaddingREM = 0.225;
+
 const notificationIconContainerStyles = memoizeFunction(
-  (): IIconStyles => ({
+  (theme: ITheme): IIconStyles => ({
     root: {
+      borderRadius: '1rem',
```

Comment:
Please use `theme.effects.roundedCorner` for this.

Reply:
We explicitly use `1rem` here because we're not a UI theme-tied border radius, but instead CSS to make a circle, i.e. these should always be the curvature and not be tied to theme:
![image](https://user-images.githubusercontent.com/2684369/156603628-8aa2310e-2bfa-478d-b764-0545bcfbcfcc.png)

I'll add a comment here to make this more clear

Reply:
Added some comments!